CONTRIBUTING/index.html

```diff
@@ -1,4182 +1,63 @@
-
 <!doctype html>
 <html lang="en" class="no-js">
   <head>
-    
-      <meta charset="utf-8">
-      <meta name="viewport" content="width=device-width,initial-scale=1">
-      
-      
-      
-      <link rel="icon" href="../assets/images/favicon.png">
-      <meta name="generator" content="mkdocs-1.2.4, mkdocs-material-8.1.9">
-    
-    
-      
-        <title>Contributing - Argo Workflows - The workflow engine for Kubernetes</title>
-      
-    
-    
-      <link rel="stylesheet" href="../assets/stylesheets/main.2b4465f4.min.css">
-      
-        
-        <link rel="stylesheet" href="../assets/stylesheets/palette.e6a45f82.min.css">
-        
-      
-    
-    
-    
-      
-        
-        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,400i,700%7CRoboto+Mono&display=fallback">
-        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
-      
-    
-    
-    <script>__md_scope=new URL("..",location),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
-    
-      
-  
-
-
-  
-  
-
-
-  <script>function gtag(){dataLayer.push(arguments)}window.dataLayer=window.dataLayer||[],gtag("js",new Date),gtag("config","G-5Z1VTPDL73"),document.addEventListener("DOMContentLoaded",function(){document.forms.search&&document.forms.search.query.addEventListener("blur",function(){this.value&&gtag("event","search",{search_term:this.value})}),"undefined"!=typeof location$&&location$.subscribe(function(e){gtag("config","G-5Z1VTPDL73",{page_path:e.pathname})})})</script>
-  <script async src="https://www.googletagmanager.com/gtag/js?id=G-5Z1VTPDL73"></script>
-
-
-    
-    
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <link rel="canonical" href="https://argo-workflows.readthedocs.io/en/latest/CONTRIBUTING/" />
+    <meta http-equiv="refresh" content="0;URL=https://argo-workflows.readthedocs.io/en/latest/CONTRIBUTING/" />
+    <title>Contributing - Argo Workflows - The workflow engine for Kubernetes - Argo Workflows - The workflow engine for Kubernetes</title>
```

Comment:
some have duplicated titles as well, this has "- Argo Workflows - The workflow engine for Kubernetes" twice

Reply:
I'll get PR up shortly.

Reply:
I got a branch with all the changes done now, so I think we should be covered. If you want to edit the script for posterity, that could be good

Reply:
Fixed in #12500